seaborn/distributions.py

```diff
@@ -1770,6 +1827,57 @@ def kdeplot(
 )
 
 
+def ecdfplot(
+    data=None, *,
+    # Vector variables
+    x=None, y=None, hue=None, weights=None,
```

Comment:
Should `y` be left in, given that `y` is not used below?

Reply:
`y` is in the signature because you can draw a plot with the variable on the y axis (and the proportion/count on the y axis) by assigning to `y`.